### PR TITLE
FH-3790: Add requirement to bumped the version into the CONTRIBUTION doc of these project

### PR DESCRIPTION
**Pull request by camilamacedo86**
JIRA: https://issues.jboss.org/browse/FH-3790

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -64,6 +64,12 @@ for it to be reviewed (Status: Dev Complete).
 * Note: if you have write access to the repository, do not directly merge pull 
   requests. Let another team member review your pull request and approve it.
 
+## Before Merge
+* Bump the version into the files; `npm-shrinkwrap.json`, `package.json`, `sonar-project.properties`, e.g. version": "3.2.0-BUILD-NUMBER"
+* Prepend your commit message with a JIRA ticket number and the text Bump Version, e.g. "FH-1234: Bump Version."
```
Comment (pb82) [inline CONTRIBUTING.md]: I think the version bump commit doesn't have to be prepended with the JIRA number, only the commites with actual changes.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -64,6 +64,12 @@ for it to be reviewed (Status: Dev Complete).
 * Note: if you have write access to the repository, do not directly merge pull 
   requests. Let another team member review your pull request and approve it.
 
+## Before Merge
+* Bump the version into the files; `npm-shrinkwrap.json`, `package.json`, `sonar-project.properties`, e.g. version": "3.2.0-BUILD-NUMBER"
```
Comment (pb82) [inline CONTRIBUTING.md]: into -> in

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -64,6 +64,12 @@ for it to be reviewed (Status: Dev Complete).
...
+* Prepend your commit message with a JIRA ticket number and the text Bump Version, e.g. "FH-1234: Bump Version."
+* Ensure that the commits into the PR are properly squashed and you have at list one commit for the changes and one for the bump version.
```
Comment (pb82) [inline CONTRIBUTING.md]: into -> in

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -64,6 +64,12 @@ for it to be reviewed (Status: Dev Complete).
...
+* Ensure that the code don't have sonar issues/comments ( E.g MENOR )
```
Comment (pb82) [inline CONTRIBUTING.md]: don't -> doesn't

**Comment by pb82** [comment]
LGTM, just a few minor corrections

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -38,7 +38,7 @@ not already exist.
 * Follow the coding style in use.
 * Check for unnecessary whitespace with `git diff --check` before committing.
 * Make sure you have added the necessary tests for your changes.
-* Run _all_ the tests to assure nothing else was accidentally broken.
+* Run _all_ the tests via `$grunt test`command to assure nothing else was accidentally broken.
```
Comment (finp) [inline CONTRIBUTING.md]: suggestion (dunno whether 'bug' or 'issue' is best word below):
Run _all_ the tests using the `$grunt test`command to make sure your change did not accidentally introduce a bug.

File: CONTRIBUTING.md (hunk 3 above)
Comment (finp) [reply CONTRIBUTING.md]: so, you need a minimum of two commits? if so, this text if fine

File: CONTRIBUTING.md (hunk 1 above)
Comment (camilamacedo86) [reply CONTRIBUTING.md]: 👍 Tks for the review.  Done the change.

File: CONTRIBUTING.md (hunk 4 above)
Comment (camilamacedo86) [reply CONTRIBUTING.md]: 👍 Done tks

File: CONTRIBUTING.md (hunk 3 above)
Comment (camilamacedo86) [reply CONTRIBUTING.md]: @finp yep 👍 tks 
@pb82 Done:-)

File: CONTRIBUTING.md (hunk 5 above)
Comment (camilamacedo86) [reply CONTRIBUTING.md]: 👍 Tks Done.

**Comment by camilamacedo86** [comment]
@pb82 and @finp tks for the help and review. All changes requested are done and its is missing now just the approval.

**Comment by camilamacedo86** [comment]
@pb82 the correction as suggested was made. Could you approve this PR now?

**Pull request merged**